lago/providers/libvirt/vm.py

```diff
@@ -543,6 +543,13 @@ def _get_qemu_kvm_path(self):
             )
 
         if not qemu_kvm_path:
+            LOGGER.warning("emulator not found %r", ET.tostring(self.caps))
+            ret, err, out = utils.run_command(['virt-host-validate', 'qemu'])
+            LOGGER.warning("virt-host-validate: %s - %s (%s)", ret, err, out)
+            ret, err, out = utils.run_command(['lscpu'])
```

Comment:
I think that we can just print a warning with an explanation about the issue, and let the user run `lscpu` etc...

Reply:
> I think that we can just print a warning with an explanation about the issue, and let the user run lscpu etc...

Yes, this is part of a debug commit, i will remote it :)

I have investigated the missing `guest` tag in `virsh capabilities` at [L1](https://www.linux-kvm.org/page/Nested_Guests#Nested_Guests):
* on `ovirt-srv10` (L0) (thanks to @duck-rh):
  * nested configuration is enabled: `/etc/modprobe.d/nested.conf` contains `options kvm-intel nested=y`
  * L1 definition contains `vmx` CPU flag (checked on `vm0053` VM, using `virsh dumpxml`)
* on L1:
  * output of `virt-host-validate` looks good:
    ````
    QEMU: Checking for hardware virtualization                                 : PASS
    QEMU: Checking if device /dev/kvm exists                                   : PASS
    QEMU: Checking if device /dev/kvm is accessible                            : PASS
    QEMU: Checking if device /dev/vhost-net exists                             : WARN (Load the 'vhost_net' module to improve performance of virtio networking)
    QEMU: Checking if device /dev/net/tun exists                               : FAIL (Load the 'tun' module to enable networking for QEMU guests)
    QEMU: Checking for cgroup 'memory' controller support                      : PASS
    QEMU: Checking for cgroup 'memory' controller mount-point                  : WARN (Mount 'memory' cgroup controller (suggested at /sys/fs/cgroup/memory))
    QEMU: Checking for cgroup 'cpu' controller support                         : PASS
    QEMU: Checking for cgroup 'cpu' controller mount-point                     : WARN (Mount 'cpu' cgroup controller (suggested at /sys/fs/cgroup/cpu))
    QEMU: Checking for cgroup 'cpuacct' controller support                     : PASS
    QEMU: Checking for cgroup 'cpuacct' controller mount-point                 : WARN (Mount 'cpuacct' cgroup controller (suggested at /sys/fs/cgroup/cpuacct))
    QEMU: Checking for cgroup 'cpuset' controller support                      : PASS
    QEMU: Checking for cgroup 'cpuset' controller mount-point                  : WARN (Mount 'cpuset' cgroup controller (suggested at /sys/fs/cgroup/cpuset))
    QEMU: Checking for cgroup 'devices' controller support                     : PASS
    QEMU: Checking for cgroup 'devices' controller mount-point                 : WARN (Mount 'devices' cgroup controller (suggested at /sys/fs/cgroup/devices))
    QEMU: Checking for cgroup 'blkio' controller support                       : PASS
    QEMU: Checking for cgroup 'blkio' controller mount-point                   : WARN (Mount 'blkio' cgroup controller (suggested at /sys/fs/cgroup/blkio))
    QEMU: Checking for device assignment IOMMU support                         : WARN (No ACPI DMAR table found, IOMMU either disabled in BIOS or not supported by this hardware platform)
    ```
  * output of `lscpu` seems fine too:
    ```
    lscpu
    Architecture:        x86_64
    CPU op-mode(s):      32-bit, 64-bit
    Byte Order:          Little Endian
    CPU(s):              2
    On-line CPU(s) list: 0,1
    Thread(s) per core:  1
    Core(s) per socket:  1
    Socket(s):           2
    NUMA node(s):        1
    Vendor ID:           GenuineIntel
    CPU family:          6
    Model:               44
    Model name:          Westmere E56xx/L56xx/X56xx (Nehalem-C)
    Stepping:            1
    CPU MHz:             1999.999
    BogoMIPS:            3999.99
    Virtualization:      VT-x
    Hypervisor vendor:   KVM
    Virtualization type: full
    L1d cache:           32K
    L1i cache:           32K
    L2 cache:            4096K
    NUMA node0 CPU(s):   0,1
    Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 syscall nx lm constant_tsc rep_good nopl cpuid pni pclmulqdq vmx ssse3 cx16 sse4_1 sse4_2 x2apic popcnt aes hypervisor lahf_lm pti tpr_shadow vnmi flexpriority ept vpid arat
    ``` 

It seems `libvirtd` log on L1 should be checked ?